test/e2e_env/kubernetes/kubernetes_suite_test.go

```diff
@@ -0,0 +1,57 @@
+package auth_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/pkg/config/core"
+	"github.com/kumahq/kuma/pkg/test"
+	"github.com/kumahq/kuma/test/e2e_env/kubernetes/env"
+	healthcheck "github.com/kumahq/kuma/test/e2e_env/kubernetes/healthcheck"
+	"github.com/kumahq/kuma/test/e2e_env/kubernetes/jobs"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func TestE2E(t *testing.T) {
+	test.RunSpecs(t, "E2E Kubernetes Suite")
+}
+
+var _ = SynchronizedBeforeSuite(
+	func() []byte {
+		env.Cluster = NewK8sCluster(NewTestingT(), Kuma1, Verbose)
```

Comment:
Why use global variable `env.Cluster` if the value will be overridden in the next `allProcessBody` func?

Reply:
Good point, I skipped the initialization if Cluster is already prepared